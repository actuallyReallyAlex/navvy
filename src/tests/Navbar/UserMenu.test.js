import React from 'react'
import UserMenu from '../../Navbar/UserMenu'
import { shallow } from 'enzyme'

const defaultProps = {}

describe('<UserMenu />', () => {
  afterEach(() => {})

  test('Should render the <UserMenu /> component.', () => {
    const component = shallow(<UserMenu {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
