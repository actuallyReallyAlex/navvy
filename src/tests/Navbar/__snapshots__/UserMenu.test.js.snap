// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserMenu /> Should render the <UserMenu /> component. 1`] = `ShallowWrapper {}`;
